storage/memory/log_storage.go

```diff
@@ -342,6 +342,22 @@ func (t *logTreeTX) GetActiveLogIDs(ctx context.Context) ([]int64, error) {
 	return t.getActiveLogIDs(ctx)
 }
 
+func (t *readOnlyLogTX) GetUnsequencedCounts(ctx context.Context) (map[int64]int64, error) {
+	t.ms.mu.RLock()
+	defer t.ms.mu.RUnlock()
+
+	ret := make(map[int64]int64)
+	for id, tree := range t.ms.trees {
+		tree.mu.RLock()
+		defer tree.mu.RUnlock()
+
+		k := unseqKey(id)
+		q := tree.store.Get(k).(*kv).v.(*list.List)
```

Comment:
Could you spell it out, please? I don't think q -> queue is easy to infer form this context.